sources/packages/backend/libs/integrations/src/services/disbursement-schedule/e-cert-calculation/e-cert-calculation-process.ts

```diff
@@ -152,13 +170,56 @@ export abstract class ECertCalculationProcess {
     }
   }
 
+  /**
+   * Checks whether the disbursement blocked notification should be created or not.
+   * @param disbursementId disbursement id.
+   * @param entityManager entity manager to execute in transaction.
+   * @returns boolean indicating if the disbursement blocked notification should be created or not.
+   */
+  private async shouldCreateDisbursementBlockedNotification(
+    disbursementId: number,
+    entityManager: EntityManager,
+  ): Promise<boolean> {
+    // Create notifications to the student and ministry when the disbursement is blocked.
```

Comment:
The comment is misleading because the method is not creating the notification.
Besides that, it may be pointless to have a general comment here if the method summary is already explaining it.